eventually/team/models.py

```diff
@@ -0,0 +1,169 @@
+"""
+Team model
+=======================
+
+This module implements class that represents the team entity.
+"""
+from django.db import models
+
+
+class Team(models.Model):
+
+    """
+    .. class:: Team
```

Comment:
![image](https://user-images.githubusercontent.com/33020662/32243841-4bee49aa-be7f-11e7-82d9-fee256f62901.png)
We'll get this as a result

Reply:
Done